src/mono/browser/runtime/rollup.config.js

```diff
@@ -259,6 +259,14 @@ if (isDebug) {
     });
 }
 
+// Always copy to the wasmbrowser template so it stays in sync
```

Comment:
We will have one more dotnet.d.ts in rep for CoreCLR soon, So I would prefer to not make it worse here.

Could we copy the file from current location in src\mono\browser\runtime\dotnet.d.ts into the template at the time when we create the template nuget ?